Godeps/Godeps.json

```diff
@@ -125,6 +120,11 @@
 			"Rev": "c8388a52ec7a328b6aee3c48d31bd7f3e0b14aeb"
 		},
 		{
+			"ImportPath": "github.com/docker/docker/api/types/volume",
```

Comment:
It's this ok? I only update the codegangsta to urfave by hand.
I'm not sure because the Godeps.json should be auto generated by `godep`.